app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -211,20 +221,37 @@ export class InviteUserModal extends Component {
       });
   };
 
-  inviteUserAndCloseModal = (closeModal) => (data) => {
-    this.inviteUser(data).then((res) => {
-      if (res && res.errorOccurred) {
-        return;
-      }
-      if (data.user.externalUser) {
-        this.props.showModalAction({
-          id: 'externalUserInvitationModal',
-          data: { email: res.email, link: res.backLink },
-        });
+  inviteUserAndCloseModal = (closeModal) => async (data) => {
+    const { selectedProject, userSuggestions } = this.props;
+    const userData = {
+      ...data,
+    };
+
+    if (!(this.props.data.isProjectSelector || !userSuggestions)) {
```

Comment:
Optional. May be destructure the `isProjectSelector` from props?

Reply:
Done.